src/main/resources/assets/create/lang/es_es.json

```diff
@@ -0,0 +1,804 @@
+
+	"block.create.adjustable_chain_gearshift.tooltip": "ADJUSTABLE CHAIN GEARSHIFT",
+	"block.create.adjustable_chain_gearshift.tooltip.summary": "_Transmite_ la _rotación_ en una línea recta y a las _Transmisiones_ _de_ _Cadena_ _Empotrada_ adyacentes. Una _señal_ _de_ _redstone_ proporcionada a este bloque controlará el tamaño de la rueda motriz que se acopla a las cadenas de transmisión.",
+	"block.create.adjustable_chain_gearshift.tooltip.condition1": "Control por Redstone",
+	"block.create.adjustable_chain_gearshift.tooltip.behaviour1": "_Sin_ una señal, las cadenas adyacentes transmitirán la _misma_ _velocidad._ Con una señal a _máxima_ _potencia,_ las cadenas adyacentes transmitirán exactamente el _doble_ _de_ _su_ _velocidad._ Cualquier cosa intermedia dará resultados entre 1-2x su velocidad.",
+
+	"item.create.belt_connector.tooltip": "BELT CONNECTOR",
+	"item.create.belt_connector.tooltip.summary": "Conecta dos o más _ejes_ con una _Correa_ _Mecánica._ Los ejes conectados tendrán exactamente la misma velocidad y dirección de rotación. La cinta puede actuar como un _transportador_ para _Objetos_ y _Entidades._",
+	"item.create.belt_connector.tooltip.control1": "Click Derecho en un eje",
+	"item.create.belt_connector.tooltip.action1": "Selecciona el eje como una polea del cinturón. Ambos ejes seleccionados tienen que _alinearse_ ya sea _Verticalmente,_ _Horizontalmente,_ o _Diagonalmente_ hacia la dirección de la Correa.",
+	"item.create.belt_connector.tooltip.control2": "Click Derecho mientras se está agachado",
+	"item.create.belt_connector.tooltip.action2": "_Reinicia_ la primera posición seleccionada para la correa.",
+
+	"item.create.goggles.tooltip": "GOGGLES",
+	"item.create.goggles.tooltip.summary": "Un par de gafas para aumentar la visión con _información_ _cinética_ útil.",
+	"item.create.goggles.tooltip.condition1": "Cuando se usan",
+	"item.create.goggles.tooltip.behaviour1": "Muestra _indicadores_ _con_ _colores_ correspondientes al _Nivel_ _de_ _Velocidad_ de un componente cinético colocado, así como _Impacto_ _de_ _Estrés_ y _Capacidad_ de los componentes individuales.",
+	"item.create.goggles.tooltip.condition2": "Al mirar el medidor",
+	"item.create.goggles.tooltip.behaviour2": "Muestra información detallada sobre la _velocidad_ o el _estrés_ de la red a la que está conectado el medidor.",
+
+	"item.create.wrench.tooltip": "WRENCH",
+	"item.create.wrench.tooltip.summary": "Una herramienta útil para trabajar en artefactos cinéticos. Se puede utilizar para _Rotar,_ _Desmantelar_ y  _Configurar_ componentes.",
+	"item.create.wrench.tooltip.control1": "Click Derecho en un bloqueo cinético",
+	"item.create.wrench.tooltip.action1": "_Gira_ los componentes hacia o en contra de la cara con la que interactuó.",
+	"item.create.wrench.tooltip.control2": "Click Derecho mientras se está agachado",
+	"item.create.wrench.tooltip.action2": "_Desmonta_ los _componentes_ _cinéticos_ y los coloca de nuevo en _tú_ _inventario._"
```

Comment:
_Desmonta_ los _componentes_ _cinéticos_ y los coloca de nuevo en _tu_ _inventario._"

Reply:
@muriplz Hello! I'm redoing the Spanish translation from scratch, this template had many errors, if you want you can check the new one

Reply:
@FrannDzs Hello! I've just seen it, I'll take a look on my free time <3

Reply:
> @FrannDzs Hello! I've just seen it, I'll take a look on my free time <3
Hello! the new translation is in the draft, forget about this PR. I still have syntax errors that won't allow the file to load but I'm finishing fixing it now, I completed everything.

Reply:
can I have the link from the new file? I thought this one was the latest, it's from 8 days ago.